### PR TITLE
fix(Investigators): Change  investogator fetch  for Test env

### PR DESCRIPTION
**Pull request by doronlnd84**
Due to perfomance issue I changed fetching investigator's fetch to be by SQL Function.
Now,  a json object of user returned from DB.
**Since DB user object is different from  User interface I had to add my own Inteface  (FetchedUser).**

this might be  tested  with DB  CoronaiTest1  which is copy of Test DB

### CODE REVIEW

**Review by eladyashar — APPROVED** [review]

File: server/src/ClientToDBAPI/UsersRoute/mainRoute.ts [hunk 1]
```diff
@@ -194,20 +194,20 @@ usersRoute.get('/group', adminMiddleWare, (request: Request, response: Response)
         user: response.locals.user.id,
     });
     groupLogger.info(`querying the graphql API with parameters ${JSON.stringify({ investigationGroup: response.locals.user.investigationGroup })}`, Severity.LOW);
-    graphqlRequest(GET_ACTIVE_GROUP_USERS, response.locals, { investigationGroup: +response.locals.user.investigationGroup })
+    graphqlRequest(GET_ACTIVE_GROUP_USERS, response.locals, { inputCountyId: +response.locals.user.investigationGroup })
         .then((result: any) => {
-            let users: User[] = [];
-            if (result && result.data && result.data.allUsers) {
```
Comment (ItayBenMoshe) [inline server/src/ClientToDBAPI/UsersRoute/mainRoute.ts:200]: why did you delete the condition?

**Review by ItayBenMoshe — APPROVED** [review]
